src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -393,11 +393,41 @@ private MobileAdsDataProvider() {
   }
 
   @DataProvider
+  public static Object[][] providersChainMercury() {
+    return new Object[][]{
+        {
+            "project43",
+            "SyntheticTests/ProvidersChain",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/ProvidersChain" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)